authorizers/iconik/headers-authorizer.ts

```diff
@@ -0,0 +1,39 @@
+import { getEnv } from '@helper/environment';
+import { errorHandler } from '@helper/error-handler';
+import { log } from '@helper/logger';
+import { IconikParams } from '@workflowwin/iconik-api';
+import { APIGatewayAuthorizerEvent, APIGatewayAuthorizerResult, Handler } from 'aws-lambda';
+import { getCallerAndOwner } from './helper';
+import { generatePolicy } from '../policy-generator';
+import { IconikContext, IconikEnhancedAuthContext } from './interfaces/context';
+
+export const iconikAuthorizer: Handler<
+  APIGatewayAuthorizerEvent,
+  (APIGatewayAuthorizerResult & { context: IconikEnhancedAuthContext }) | undefined
+> = async (event, context) => {
+  log('[Iconik Authorizer]', event);
+  try {
+    const callerId: string = event['headers']['User-Id'];
+    const authToken: string = event['headers']['auth-token'];
+
+    const iconikParams: IconikParams = {
+      authToken,
+      iconikUrl: getEnv('ICONIK_URL'),
+      appId: getEnv('ICONIK_APP_ID'),
+      systemDomainId: getEnv('ICONIK_DOMAIN_ID'),

```

Comment:
Move these variables like this 
![image](https://user-images.githubusercontent.com/26054907/116522000-54dde880-a8dd-11eb-9620-a72bb6de1270.png)